packages/circuits/semaphore.circom

```diff
@@ -4,22 +4,64 @@ include "babyjub.circom";
 include "poseidon.circom";
 include "binary-merkle-root.circom";
 
+// The Semaphore circuit can be divided into 3 main parts.
+// The first part involves the generation of the Semaphore identity,
+// i.e. the public key and its hash, which is called the commitment
```

Comment:
"... which is called the commitment ...". Can be confusing: public key or hash or both of them?

Reply:
The hash, "which" is after "hash,", so it should be pretty clear